src/python/grpcio_test/grpc_interop/_secure_interop_test.py

```diff
@@ -45,19 +48,19 @@ class SecureInteropTest(
     unittest.TestCase):
 
   def setUp(self):
-    self.server = implementations.server(
-        methods.SERVICE_NAME, methods.SERVER_METHODS, 0,
-        private_key=resources.private_key(),
-        certificate_chain=resources.certificate_chain())
+    self.server = test_pb2.beta_create_TestService_server(methods.TestService())
+    port = self.server.add_secure_port(
+        '[::]:0', implementations.ssl_server_credentials(
+            [(resources.private_key(), resources.certificate_chain())]))
     self.server.start()
-    port = self.server.port()
-    self.stub = implementations.stub(
-        methods.SERVICE_NAME, methods.CLIENT_METHODS, 'localhost', port,
-        secure=True, root_certificates=resources.test_root_certificates(),
-        server_host_override=_SERVER_HOST_OVERRIDE)
+    self.stub = test_pb2.beta_create_TestService_stub(
+        test_utilities.not_really_secure_channel(
+            '[::]', port, implementations.ssl_client_credentials(
```

Comment:
I'm not used to seeing '[::]' used for the host in channel construction. I suppose it works? Is it more correct than "localhost"?